Tests/UnitTests/fss-search.component.spec.ts

```diff
@@ -140,10 +141,10 @@ export const attributeSearchFilterMockData: any =
          "expanded": true
       },
       {
-         "title": "weeknumber",
+         "title": "Year / Week",
```

Comment:
Please check if `attributeSearchFilterMockData` is actually used. If not, please delete it.

Reply:
Resolved